src/components/GalaxyModal.js

```diff
@@ -324,7 +312,7 @@ export const GalaxyModalBase = withDisplayName('GalaxyModal')(
                         id,
                         value: dataDisk.diskType,
                         isDisabled: true,
-                        onChange: value => setDataDisk(_.set('diskType', value.value)),
+                        onChange: value => setDataDisk(_.set('diskType', value)),
```

Comment:
I'd suggest adding a helper for disk size/type now that it's an object, similarly to 

```javascript
  const updateComputeConfig = _.curry((key, value) => setComputeConfig(_.set(key, value)))
```

in `ComputeModal`. With that, we should be able to have the following

```suggestion
                        onChange: updateDisk('type'),
```